From 70d098b23730ddead9cf1d23eeda4c83e32cd878 Mon Sep 17 00:00:00 2001
From: Petr Viktorin <pviktori@redhat.com>
Date: Tue, 5 Feb 2013 06:47:30 -0500
Subject: [PATCH 119/119] Don't add another nsDS5ReplicaId on updates if one
 already exists

Modify update file to use default: rather than add: in
cn=replication,cn=etc,$SUFFIX.

Drop quotes around nsDS5ReplicaRoot because default: values
are not parsed as CSV.

https://fedorahosted.org/freeipa/ticket/3394
---
 install/updates/20-replication.update | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/install/updates/20-replication.update b/install/updates/20-replication.update
index 29823a6faa5eb95e8d46a393035a6ae7ff56a8b8..891cbcb881120028ee411caaa4aa79096c3ed589 100644
--- a/install/updates/20-replication.update
+++ b/install/updates/20-replication.update
@@ -4,6 +4,6 @@
 # Start at 3 to avoid conflicts with v1.0 replica ids. The value itself
 # isn't important but each replica needs a unique id.
 dn: cn=replication,cn=etc,$SUFFIX
-add: objectclass: nsDS5Replica
-add: nsDS5ReplicaId: 3
-add: nsDS5ReplicaRoot: '$SUFFIX'
+default: objectclass: nsDS5Replica
+default: nsDS5ReplicaId: 3
+default: nsDS5ReplicaRoot: $SUFFIX
-- 
1.9.3

